tools/test-backend

```diff
@@ -62,7 +62,7 @@ not_yet_fully_covered = [
     "corporate/lib/activity.py",
     "corporate/lib/remote_billing_util.py",
     # Major lib files should have 100% coverage
-    "zerver/lib/addressee.py",
+    "zerver/actions/reactions.py" "zerver/lib/addressee.py",
```

Comment:
This is very broken; you're generating `zerver/actions/reactions.pyzerver/lib/addressee.py` here.